index.html

```diff
@@ -9,11 +9,11 @@
   <body>
     <div>
       <header>
-        <h2 class="studentName">Student Page</h2>
+        <h2 class="studentName">bara'a</h2>
       </header>
 
       <section class="container">
-        <h3 class="aboutMe">About me</h3>
+        <h3 class="aboutMe">Im student in ppu</h3>
```

Comment:
Great Job Bara! :star_struck:  
Just make sure to add the details about you on a `<p>` tag not an `<h3>`.
for example:
```html
<section class="container">
    <h3 class="aboutMe">About me</h3>
    <p>your details</p>
</section>
```